src/python/WMComponent/TaskArchiver/CleanCouchPoller.py

```diff
@@ -982,6 +1008,7 @@ def getPerformanceFromDQM(self, dqmUrl, dataset, run):
         except Exception as ex:
             logging.info("Actually got a JSON from DQM perf in for %s run %d , but content was bad, Bailing out",
                          dataset, run)
+            logging.exception(ex)  # Let's print the stacktrace with generic Exception
```

Comment:
maybe you better cast it to string, thus `logging.exception(str(ex))`

Reply:
good point

Reply:
done